tests/TestCase.php

```diff
@@ -74,7 +74,7 @@ protected function resolveApplicationConfiguration($app)
         ];
 
         foreach ($configs as $config) {
-            $app['config']->set("statamic.$config", require(__DIR__."/../config/{$config}.php"));
+            $app['config']->set("statamic.$config", require (__DIR__."/../config/{$config}.php"));
```

Comment:
This is unrelated to this PR. It's a new Pint thing but since it's one line I'll just let it slide. Avoids a whole PR just for one character. 🤫